conf/messages.cy

```diff
@@ -114,8 +114,8 @@ confirmation_view.viewClientAccount = Yn ôl i fanylion y cleient
 
 incorrect_mandation.heading = Mae’r busnes yn cyflwyno Ffurflenni TAW drwy ddefnyddio’r cynllun Troi Treth yn Ddigidol
 incorrect_mandation.paragraph = Mae’r busnes wedi cofrestru ar gyfer y gwasanaeth Troi Treth yn Ddigidol.
-incorrect_mandation.agent.text = Mae’n rhaid i chi gyflwyno Ffurflenni TAW eich cleient i CThEM gan ddefnyddio meddalwedd gyfrifyddu sy’n cydweddu.
-incorrect_mandation.nonAgent.text =  Mae’n rhaid i chi gyflwyno’ch Ffurflenni TAW i CThEM gan ddefnyddio meddalwedd gyfrifyddu sy’n cydweddu.
+incorrect_mandation.agent.text = Mae’n rhaid i chi gyflwyno Ffurflenni TAW eich cleient i CThEFgan ddefnyddio meddalwedd gyfrifyddu sy’n cydweddu.
```

Comment:
Can you add the space between CThEF and gan please. This needs to be done for both entries.

Reply:
Added space